src/components/DashboardHeader.tsx

```diff
@@ -72,7 +72,7 @@ const DashboardHeader = () => {
             {t('sidebar.contact')}
           </Button>
         </div>
-        <Tooltip title={t('sidebar.profile')}>
+        <Tooltip title={<>{t('sidebar.profile')}</>}>
```

Comment:
Dlaczego w `Tooltip` prop `title` przyjmuje `JSX.Element` a nie `string`?
To nie ma żadnego sensu...

Reply:
title w Tooltipie nie przyjmuje stringa. Tu jest output z typescripta:

_TypeScript error in /home/marcin/Documents/revolve/tfa/osmintegrator-frontend/src/components/DashboardHeader.tsx(86,18):
Type 'TFunctionResult' is not assignable to type 'boolean | ReactChild | ReactFragment | ReactPortal'.
  Type 'undefined' is not assignable to type 'boolean | ReactChild | ReactFragment | ReactPortal'._

Reply:
Wyrzucało mi błąd, w którym była informacja, ze powinien to być React Fragment, bądź React Child, co mnie zmyliło. Wystarczyło dodać  `<Tooltip title={t<string>('logout.button')}>` i problem zniknął.